htdocs/xoops_lib/Xoops.php

```diff
@@ -1046,7 +1046,7 @@ public function getModuleForm($obj, $name, $module_dir = null)
             }
         }
         if (XoopsLoad::fileExists(
-            $hnd_file = \XoopsBaseConfig::get('root-path') . "/modules/{$module_dir}/class/form/{$name}.php"
+            $hnd_file = \XoopsBaseConfig::get('root-path') . '/modules/'.$module_dir.'/class/form/'.$name.'.php'
```

Comment:
I don't get this. The two lines should evaluate to the exact same thing.

What were the values of $module_dir and $name that caused an issue?